src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -369,16 +302,115 @@ function GardenMetadata() {
             css="width: 100%;"
           />
         </Field>
+
+        <div
+          css={`
+            width: 100%;
+          `}
+        >
+          <div>
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+              `}
+            >
+              {/* We need to have this Field without using the field component to avoid the  issue about clicking outside  the file chooser and poping up the selelector */}
+              <span
+                css={`
+                  ${textStyle('body2')};
+                  color: ${theme.content};
+                  margin-right: ${0.5 * GU}px;
+                `}
+              >
+                ASSETS (optional)
+              </span>
+              <Help>
+                If you don’t have this images yet, you will get default ones
+                assigned in the meantime.
+              </Help>
+            </div>
+            <div
+              css={`
+                display: flex;
+                flex-direction: column;
+                ${textStyle('body2')};
+                color: ${theme.contentSecondary};
+                margin-top: ${2 * GU}px;
+              `}
+            >
+              <span>Drag and drop or browse your files to upload one.</span>
+              <span
+                css={`
+                  color: ${formatValidationColor};
+                `}
+              >
+                Valid file formats are: JPG and PNG
+              </span>
+            </div>
+          </div>
+          <div
+            css={`
+              margin-top: ${2 * GU}px;
+              display: flex;
+              align-items: center;
+              justify-content: space-between;
+            `}
+          >
+            <div
+              css={`
+                min-width: 194px;
+              `}
+            >
+              <FileUploaderField
+                allowedMIMETypes={['image/jpeg', 'image/png']}
+                file={formData.logo_type}
+                onDragaAccepted={handleOnDragAccepted}
+                onDragRejected={handleOnDragRejected}
+                onFileUpdated={handleOnGardenLogoTypeUpdated}
+                label="GARDEN LOGO TYPE"
```

Comment:
```suggestion
                label="GARDEN LOGOTYPE"

Reply:
I think the actual name of this kind of asset is logotype. I wonder if we should explain where is going to be displayed or have a similar help tip with an image of how it will look.

Reply:
mmm yeah not sure this name logotype also doesn't  make to much sense to me to what is representing, so i think we just need to change the naming to fully represent  what  they are